### DISCUSSION TITLE
Increase max length of Taxon `source` field

### DISCUSSION THREAD

**Discussion by alesan99** [specify/specify7#5973] [Feature Requests]
Many records from The Catalogue of Life have sources that exceed the current limit of 64 characters. I've encountered ones as long as 928 characters.

This causes issues whenever importing a taxa tree so the only option is to use another text field for the source.

**Comment by grantfitzsimmons** [comment]
I concur ^ 👏

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/7480
